pom.xml

```diff
@@ -835,6 +845,12 @@
                 <version>${project.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>${project.groupId}</groupId>
+                <artifactId>wildfly-microprofile-opentracing-smallrye</artifactId>
+                <version>${project.version}</version>
```

Comment:
This and all other compile scope dependencies should have exclusions

Reply:
May I ask you to confirm that this should happen *here*, at the root pom? It looks like the other dependencies listed here don't have any exclusions. The `feature-pack/pom.xml` do have them, though.

Reply:
@jamezp @stuartwdouglas @bstansberry WDYT?

When I looked quickly earlier I saw some dependencies having exclusions. A closer look shows it isn't totally consistent, so there are other dependencies which don't have exclusions.

Reply:
I think we've gone about it a couple different ways. I'm personally of the feeling that third-party dependencies should exclude all dependencies and include any implicit required dependencies. However for dependencies that are a maven module of the project I think it's okay to not exclude all dependencies.

Reply:
Could someone please confirm which one should I use here in the main pom? Looks like excluding everything is not a rule. In fact, most dependencies on this pom are *not* excluding everything.

I would not mind excluding everything, I just want to follow what's the common practice.